### DISCUSSION TITLE
V3.5.0 Add NAPALM Driver for Plugin netbox_napalm_plugin fails

### DISCUSSION THREAD

**Discussion by willikubny** [netbox-community/netbox#12389] [Help Wanted!]
With the new version 3.5.0, I've tried to add NAPALM which is now a plugin.
Setup netbox-docker with the netbox_napalm_plugin works but the in the UI when I try to add a NAPALM driver I get the following error message.

<img width="829" alt="Screenshot 2023-04-28 at 16 24 54" src="https://user-images.githubusercontent.com/70367776/235175221-98332ce4-1fbc-4669-b3ea-ef2bd2811ce3.png">

Any idea how I solve this is very appreciated.

**Comment by abhi1693** [comment]
For reference: https://github.com/netbox-community/netbox-napalm/issues/7
